packages/world-modules/src/modules/erc20-puppet/registerERC20.sol

```diff
@@ -18,13 +18,14 @@ import { ERC20Registry } from "./tables/ERC20Registry.sol";
  */
 function registerERC20(
   IBaseWorld world,
+  address puppetModule,
```

Comment:
i feel like this additional params is making it pretty impractical to use since it's currently not easy to find the address of the puppet module. Why don't we add an overload for this function that doesn't require the puppetModule address and instead calls this overload with the CREATE2 default address of the puppet module? That way it's not a breaking change for consumers (on CREATE2 compatible chains) but still allows customisation if necessary.

Reply:
The puppet module is part of the deployment pipeline right now, so doesn't use `Create2`. For example Sky Strife manually deploys the puppet: https://github.com/latticexyz/skystrife-public/blob/main/packages/contracts/script/PostDeploy.s.sol#L111-L119